### PR TITLE
Added Inflector and WebServerBundle labels

### PR DESCRIPTION
**Pull request by wouterj**

### CODE REVIEW

**Comment by chalasr** [comment]
What about adding DotEnv btw?

**Comment by wouterj** [comment]
Sounds good, but `DotEnv` doesn't yet have a label in `symfony/symfony`, so it can't be in the valid label list.

**Comment by sstok** [comment]
When the label does not exist GitHub registers it 😆 but only for the API btw.

**Comment by xabbuh** [comment]
The `Dotenv` label is now present in both repositories symfony/symfony and symfony/symfony-docs.

**Comment by wouterj** [comment]
Ready to merge

File: src/AppBundle/Subscriber/AutoLabelPRFromContentSubscriber.php [hunk 1]
```diff
@@ -109,7 +109,7 @@ private function getValidLabels()
             'Asset', 'BC Break', 'BrowserKit', 'Bug', 'Cache', 'ClassLoader',
             'Config', 'Console', 'Critical', 'CssSelector', 'Debug', 'DebugBundle',
             'DependencyInjection', 'Deprecation', 'Doctrine', 'DoctrineBridge',
-            'DomCrawler', 'Drupal related', 'DX', 'Easy Pick', 'Enhancement',
+            'DomCrawler', 'DotEnv', 'Drupal related', 'DX', 'Easy Pick', 'Enhancement',
```
Comment (xabbuh) [inline src/AppBundle/Subscriber/AutoLabelPRFromContentSubscriber.php]: The name in fact is `Dotenv` (with a lowercased `e`).

Comment (HeahDude) [inline src/AppBundle/Subscriber/AutoLabelPRFromContentSubscriber.php]: This PR should be merged with the upcoming 3.3 release, and we also need "Image", "Lock" and "UTF-8" :).

Comment (chalasr) [inline src/AppBundle/Subscriber/AutoLabelPRFromContentSubscriber.php]: @HeahDude None of them for now :)

Comment (HeahDude) [inline src/AppBundle/Subscriber/AutoLabelPRFromContentSubscriber.php]: @chalasr Actually components and labels exist so Carson should be aware of them even if they are only in master to be able to do its job ;)

**Pull request merged**